test/lib/usd/translators/testUsdExportAnimation.py

```diff
@@ -0,0 +1,64 @@
+#!/pxrpythonsubst
```

Comment:
Sorry @dgovil, one more thing for you. :)

I think this newly added test got left out of the CMakeLists.txt?

Reply:
Yes, you're right, and thanks for catching that. I'd missed cherry picking that out of our internal repo.
Fixed in: https://github.com/Autodesk/maya-usd/pull/1012

Reply:
I would like us to have more visibility on code coverage. This way we could automatically detect if PR is lowering the coverage. 

Of course, this should have been caught easily by checking logs, but I would like to eliminate possibility for errors and have some metrics to indicate how well pull request changes are tested.

Reply:
Yeah that's a good idea. It would have to be two fold though, because in this case the test existed so coverage tools might erroneously consider it covered?

Perhaps a test to see if new test files are covered in the CMakeList, since it's an easy mistake to make would help? I assume the project prefers not to just Glob all the tests in the dir right?

Reply:
Have a look for example at lcov (and gcov) to get a better sense of what type of information you can acquire from your test suit. And maybe it wasn’t explicit, but test actually have to run in order to report coverage for new code blocks/methods.

Reply:
Ah I misunderstood what you were saying. Yes that makes sense.